src/main/java/com/iota/iri/controllers/TransactionViewModel.java

```diff
@@ -863,6 +863,30 @@ public void isMilestone(Tangle tangle, Snapshot initialSnapshot, final boolean i
         }
     }
 
+    /**
+     * This method sets the {@link Transaction#conflicting} flag.
+     *
+     * It first checks if the {@link Transaction#conflicting} flag has changed. If so, it issues a database update.
+     * @param tangle            Tangle instance
+     * @param initialSnapshot   The snapshot representing the start of the ledger
+     * @param isConflicting     True if the transaction is conflicting and ignored during balance computation
+     * @throws Exception        If something goes wrong
+     */
+    public void isConflicting(Tangle tangle, Snapshot initialSnapshot, final boolean isConflicting) throws Exception {
+        if(isConflicting != transaction.conflicting){
+            transaction.conflicting = isConflicting;
+            update(tangle, initialSnapshot, "conflicting");
+        }
+    }
+
+    /**
+     * Checks if a transaction is empty.
+     * @return True if empty. False otherwise.
+     */
+    public boolean isEmpty(){
+        return Arrays.equals(getBytes(), new byte[SIZE]);
+    }
+
```

Comment:
Currently we are just check for `TransactionViewModel.PREFILLED_SLOT`
I am not saying that this is smart (I didn't make this weird design)

But it will be more awful if we have more than one way to do this check...
So I wouldn't have this method now